src/pages/add-kit-model/add-kit-model.ts

```diff
@@ -30,14 +33,31 @@ export class AddKitModelPage {
   ngOnInit() {
     this.brandsActions.fetchBrands();
     this.modelsActions.fetchModels();
+    this.action = this.navParams.get('action');
+
+    // Fill the array of options for selecting the quantity
+    this.numbers = Array.from({ length: 100 }, (value, index) => index + 1);
```

Comment:
This could eventually support dynamically setting the available quantity based on the items in the inventory. Nice that we have an easy way to add that feature in!